apps/teams-test-app/e2e-test-data/authentication.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Authentication",
+  "platforms": "*",
+  "testCases": [
+    {
+      "title": "getUser API Call - Success",
+      "version": "1.x || >2.0.0-beta.2",
+      "type": "callResponse",
+      "platformsExcluded": ["iOS"],
+      "boxSelector": "#box_getUser",
+      "expectedTestAppValue": "{\"oid\":\"mockoid\",\"tid\":\"mocktid\",\"upn\":\"mockupn\",\"loginHint\":\"mockLoginHint\",\"displayName\":\"mockName\",\"dataResidency\":\"public\"}"
+    },
+    {
+      "title": "getUser API Call - Success",
+      "version": "2.0.0-beta.2",
+      "type": "callResponse",
+      "platformsExcluded": ["iOS"],
```

Comment:
Why there is a need to include platformExcluded ? Can't platforms field handle it ? Just curious is auth is not supported on iOS?

Reply:
there is a one for capability-wide control and what you labelled here is for case-wide control. Saying maybe for a capability there are 20 cases and each of them is driven by corresponding data. But there might be one or two cannot be used for iOS/Android platform. Thus, we need another field in each case to precisely exclude it for a specific platform.

Auth is supported in iOS. Same as implemented in web sdk, most of auth capability E2E test cases are implemented directly on the platform. And the data of getUser case here is excluded because the user info provided from web is different from the one in iOS and web side might need to change to get aligned with that in iOS so I excluded it temporarily.